src/Umbraco.Configuration/Models/RuntimeSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Extensions.Configuration;
+using Umbraco.Core.Configuration;
+
+namespace Umbraco.Configuration.Models
+{
+    internal class RuntimeSettings : IRuntimeSettings
+    {
+        private readonly IConfiguration _configuration;
+        public RuntimeSettings(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public int? MaxQueryStringLength => _configuration.GetValue<int?>("Umbraco:CMS:Runtime:MaxRequestLength");
```

Comment:
Not sure where we even use these in the CMS? seems like we don't need these.

Reply:
Currently used by ClientDependency